src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Subscriber.kt

```diff
@@ -153,7 +146,7 @@ class SubscribeManager(subscriptionId: String,
     }
 
     fun pushMessage(taskId: String, rootFileName: String, zipBytes: ByteString) {
-        (this.receiver as TaskReceiver).processCompileTask(taskId, rootFileName, zipBytes)
+        (this.receiver as TaskReceiver).unzipCompileTask(taskId, rootFileName, zipBytes)
```

Comment:
If you cast it anyway then why don't you just declare its exact type in the constructor of `SubscribeManager`?

Reply:
Changed type of receiver, but next time I need to manage other Receiver.